docs/deployment.md

```diff
@@ -76,3 +76,27 @@ Note it is http not https
 #### updating a running instance
 If you want to update an already running instance with a different version of the application, you need
 to ensure that the image is created and pushed to gcr.io using `make push-image-gcp` after which you can update an instance to use the image.
+
+
+#### Auto deployment
+The auto deployment is done using CI and CD configuration.Every push on github is tested before being merged to develop.It is then deployed to staging servers and thereafter to production automatically.
```

Comment:
Currently auto-deployment happens in staging server only

Reply:
Okay correcting it